pkgs/applications/networking/flent/default.nix

```diff
@@ -1,4 +1,33 @@
-{ stdenv, buildPythonApplication, fetchFromGitHub, matplotlib, netperf, procps, pyqt5 }:
+{ stdenv, buildPythonApplication, fetchFromGitHub, matplotlib, netperf, procps, pyqt5
+, makeWrapper, iperf2
+, withMan ? true, sphinx ? null
```

Comment:
I don't think it's worth making the manual optional. Doesn't take long to build (and hydra does it for you anyway), and it's put in a separate output. So I suggest to remove this option and always build the manual, which is the de-facto standard in nixpkgs.

Reply:
I received different feedback on other PRs but I definitely agree we should encourage manpages so I build it by default.